app/shared/detectLanguage.ts

```diff
@@ -0,0 +1,6 @@
+import franc from 'franc';
+import get from 'shared/languagesList';
+
+export default {
```

Comment:
do not use export default for new files, i think is time we add https://github.com/import-js/eslint-plugin-import/blob/master/docs/rules/no-default-export.md to our rule set as a warning maybe ?

Reply:
changed, I did it export default to keep it as it was in the previous file.